test/e2e/cypress/integration/profiles/recovery/recovery/errors.spec.ts

```diff
@@ -122,6 +122,17 @@ context('Account Recovery Errors', () => {
           cy.noSession()
         })
       })
+
+      it('invalid remote recovery email template', () => {
+        const identity = gen.identityWithWebsite()
```

Comment:
We wanted to add a test which writes to the config file that then changes the schema, to ensure the hot reloading works.

Reply:
So we want to update the config on the fly here inside this test? So instead of having the courier configs added to the already generated kratos configs you want it added here?

Reply:
Exactly, that would catch resolve any future regressions :) If you need help with the code LMK!